src/server/serverless/bulk-qrcode-generation/qrCode.js

```
@@ -0,0 +1,204 @@
+const cheerio = require('cheerio')
+const fs = require('fs')
+const QRCode = require('qrcode')
+
+const { resolve } = require('path')
+const sharp = require('sharp')
+const JSZip = require('jszip')
+
+const IMAGE_WIDTH = 1000
+const QR_CODE_DIMENSIONS = 800
+const MARGIN_VERTICAL = 85
+const FONT_SIZE = 32
+const LINE_HEIGHT = 1.35
+
+const ImageFormat = {
+  SVG: 'image/svg+xml',
+  PNG: 'image/png',
+  JPEG: 'image/jpeg',
+}
+
+const FileExtension = {
+  SVG: 'svg',
+  PNG: 'png',
+  JPEG: 'jpeg',
+}
+
+const envVariant = process.env.ASSET_VARIANT
+
+const domain = process.env.DOMAIN_URL
+
+const assetVariants = ['gov', 'edu', 'health']
+if (!assetVariants.includes(envVariant)) throw Error('Invalid variant name!')
```

should also throw error if domain is missing